### PR TITLE
Feature/frontend

### PR DESCRIPTION
**Pull request by NoTengoBattery**
This branch contains the complete frontend for the clubhouse project.
- We use bootstrap to style all the views for the post (create, edit) and user registration(sign in, sign up)
- We also style the errors so they can be displayed in a better way.
- We introduce a navbar just so the app looks more appealing  
- We use the devise gem to handle user authentication and more stuff related to the user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+# ClubHouse
+This project is a Clubhouse app. The idea is to create an app where the members of the Club can publish posts anonymously. Everyone outside the club should be able to see those posts but not the name of the author unless they are members of the Club.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Getting Started :
+- Clone the repository: `git clone` https://github.com/NoTengoBattery/club-house.git
+- Move inside the club-house folder: `cd club-house`
+- Run the server: `rails s` or `rails server`
+- To see the app on your local machine open your browser and type: `localhost:3000`. Have fun !!!
```
Comment (GzimAsani) [inline README.md:12]: Tell your users that they should run rails db:create and rails db:migrate

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle: 
**Really good job with this project :muscle:  I think it looks really good and it has great functionality. But I have some changes that I want you guys to make**:
- [x] The application routes should be `resources :blogs, only: [:new, :create, :index]` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)). Mainly you should specify only the useful routes.
- [x] Your `def new` posts_ controller should initialize an empty post @post = Post.new ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_new_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))
- [x] There should be a before_action that restrict access to the :new and :create methods 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))
- [OPTIONAL] Add pictures of your app in your README, it will look more professional :+1: 

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

File: README.md [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
-application up and running.
+## App Home Page
+![](app/assets/images/screenshot.png)
 
-Things you may want to cover:
+## Getting Started :
+- Clone the repository: `git clone` https://github.com/NoTengoBattery/club-house.git
+- Move inside the club-house folder: `cd club-house`
+- Run the server: `rails s` or `rails server`
+- Run: `rails db:create` and `rails db:migrate`.
```
Comment (enaburekhan) [inline README.md]: Run rails db:create and rails db:migrate should come before Run the server: rails s or rails server.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Required changes:full_moon:
Hi team!
Great job on implementing almost all the changes requested by the previous Tse. Your effort is highly commendable:clap::clap:. Just a few changes we still need to make to get this project approved:muscle:.

**Mandatory requirements**

**PostsController**
 

- [x] it uses `before_action` to restrict access to the `:new` and `:create` methods (uses ApplicationController helper method) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)).

Create a signed_in_only! method inside the helpers directory. Then use the before_action to restict it to only new and create in PostsController.

**Views**

- [x] index

- the author is displayed only if a user is signed in (uses ApplicationController helper method) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_index_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)).
Move the logic in index.html.erb to the helper.

- [x] As already highlighted in the PR below, correct the order of running the app in the getting started in readme.

Make the above-required changes and when you are ready, request another review.
kindly drop your questions here and ping me on slack @ Eric Enaburekhan to respond.
Happy Coding
Eric Enaburekhan

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge !!!

**Pull request merged**